docs/superpowers/plans/2026-05-19-kover-coverage.md

```
@@ -0,0 +1,785 @@
+---
+
+### Task 7: Add coverage badge to README
+
+**Files:**
+- Modify: `README.md`
+
+- [ ] **Step 1: Add badge placeholder**
+
+In `README.md`, line 10 currently reads (everything on one line):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)